src/plonk/verifier.rs

```diff
@@ -52,10 +51,8 @@ impl<'a, C: CurveAffine> Proof<C> {
         let x_1: C::Scalar = get_challenge_scalar(Challenge(transcript.squeeze().get_lower_128()));
```

Comment:
it seems asymmetric that `x_0`, `x_1` are sampled within the permutation argument on the prover's side, but within the main PLONK argument on the verifier's side.